src/messages/web-compat/web-share.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "WebShare",
+  "additionalProperties": false,
+  "description": "todo: add description for `webShare` message",
+  "required": ["method", "params", "result"],
+  "properties": {
+    "method": {
+      "const": "webShare"
+    },
+    "params": {
+      "title": "WebShareParams",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "title": {
+          "description": "todo: add description for 'title' field",
+          "type": "string"
+        },
+        "url": {
+          "description": "todo: add description for 'url' field",
+          "type": "string"
+        },
+        "text": {
+          "description": "todo: add description for 'text' field",
+          "type": "string"
+        }
+      }
+    },
+    "result": {
+      "description": "todo: add return type here"
+    }
+  }
+}
```

Comment:
This is the proposed format - the naming of `method`, `params` + `result` are based on the documented keys in https://duckduckgo.github.io/content-scope-scripts/modules/Messaging.html

Reply:
Sorry, the link was incorrect, it's more like these: https://duckduckgo.github.io/content-scope-scripts/classes/Messaging_Schema.NotificationMessage.html

in general, we followed the JSON RPC spec for the naming scheme in the message format that actually get's sent/received so this is just a way of being consistent when a feature author creates individual messages.

For example, `method`, `params`, & `result` all come from https://duckduckgo.github.io/content-scope-scripts/classes/Messaging_Schema.RequestMessage.html